### PR TITLE
Updated file download to use native browser download.

### PR DESCRIPTION
**Pull request by MillenniumFalconMechanic**

### CODE REVIEW

File: spa/src/app/files/hca-download-file/hca-download-file.component.html [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+            <mat-spinner *ngIf="showSpinner(downloadResponse)"
+                         class="hca-spinner"
+                         [diameter]="14"
+                         [mode]="indeterminate">
+            </mat-spinner>
+            <ng-container *ngIf="isDownloadFailed(downloadResponse)"><div class="error">x</div></ng-container>
+            <img *ngIf="isDownloading(downloadResponse)"
+                 src="assets/images/icon/check-black.png">
+        </ng-container>
+        <a download class="hidden" #download></a>
```
Comment (hannes-ucsc) [inline spa/src/app/files/hca-download-file/hca-download-file.component.html]: So the content disposition header can be embedded into the signed S3 URL. We could try to convince the DSS people to do this during checkout. We could also (very hacky) parse the signed URL returned by DSS for the object key and resign it with the header present.

However, the first thing I propose we try is actually set the `download` attribute to the file name:

```suggestion
        <a download="{{file.fileName}}" class="hidden" #download></a>
```

Comment (hannes-ucsc) [reply spa/src/app/files/hca-download-file/hca-download-file.component.html]: References:

https://davidwalsh.name/download-attribute
https://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectGET.html
https://github.com/boto/boto3/issues/610#issuecomment-443878760

Comment (hannes-ucsc) [reply spa/src/app/files/hca-download-file/hca-download-file.component.html]: @NoopDog

Comment (NoopDog) [reply spa/src/app/files/hca-download-file/hca-download-file.component.html]: Ok thanks will give this a try.

**Pull request merged**